manifests/firefox-manifest.json

```diff
@@ -35,6 +35,13 @@
     }
   ],
 
+  "background": {
+    "scripts": [
```

Comment:
We should play safe and always include `vendor/browser-polyfill.js` before loading other scripts.